eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -4,7 +4,10 @@ parameters:
   DeleteAfterHours: 8
   Location: ''
   SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+  ServiceConnection: not-specified
   ResourceType: test
+  UseFederatedAuth: true
```

Comment:
I think we'll want this to default to false to start?

Reply:
I think it should be on by default and turned it off in `tests.yml` for services which cannot migrate.

We'd start with your inventory of services and if service owners spot failures they can turn it off themselves.

Reply:
Eventually, but it means that we basically can't merge this in to the repos right now without coordinating a massive yaml change. If we set it to false then we can start to migrate one by one, and then once we reach a migrated state with exceptions, we can invert the parameter.